### ISSUE TITLE
Epic: Add computed/rollup fields to API responses

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#2337]
```[tasklist]
### Tasks
- [ ] #2308
- [ ] https://github.com/GSA-TTS/FAC/issues/2475
- [ ] https://github.com/GSA-TTS/FAC/issues/2469
- [ ] https://github.com/GSA-TTS/FAC/issues/2474
- [ ] https://github.com/GSA-TTS/FAC/issues/2473
- [ ] https://github.com/GSA-TTS/FAC/issues/2472
- [ ] https://github.com/GSA-TTS/FAC/issues/2470
- [ ] https://github.com/GSA-TTS/FAC/issues/2471
- [ ] https://github.com/GSA-TTS/FAC/issues/2491
```

**Comment by stucka** [comment]
Any chance of getting `REPORTABLECONDITION_MP` and `MATERIALWEAKNESS_MP` back in? I think `typereport_mp` is rolled up from them. See https://github.com/GSA-TTS/FAC/issues/2314

**Comment by jadudm** [comment]
Probably. (That's not a dodge.)

I have three fields I know about that MUST come through. We'll then ticket the rollups as enhancements, and try and get them in shortly as well.

I need to restructure this epic into two, so that the MUST fixes for fields that are missing are in one, and the enhancements/rollups are in another.

But, I think it's reasonable/doable. It might just take a moment.

**Comment by phildominguez-gsa** [comment]
Research doc: https://docs.google.com/document/d/1JF113c52Ywx0BK4vZQN9xKmW22hpuqJQ4PqdCunBLac/edit

**Comment by jadudm** [comment]
@stucka , we've updated #2337 and #2414 in collaboration with Federal partners, and welcome your eye here, too. Part of the challenge is mapping the historical collection(s) through to the new system. Thoughts?

**Comment by stucka** [comment]
@jadudm , I'm be glad to help out as I can but I don't know how I can get involved ... ? The three API sites I have access to have the unpatched JSON schema and no historical data to QA against.

**Comment by jadudm** [comment]
Fair enough. We won't have historical data in our system for a while yet.

**Comment by jadudm** [comment]
We have a few booleans to add (friends of `is_multiple_ueis`), but we think the API is complete and stable. We are committed to making sure we disseminate everything collected, and will follow our update/bug-fix process if we manage to discover anything that did not make it from intake through to dissemination.

I *think* that rollups will not be included at this time. We've got docs that we'll roll on fac.gov for how to compute all the rollups from the API. If we included rollups, we would be including duplicative data, and that does not feel like a good API design to the team. Given that most rollups can be computed from a single API query, we think that documenting the semantics of the rollups, with example code, is the right path to take at this time.

We also have a data dictionary mapping the historic fieldnames to the API fieldnames that will go to the static site shortly.

Both of these should hit the website shortly.
